sal_interfaces/DIMM/DIMM_Events.xml

```diff
@@ -25,6 +25,28 @@
       <Count>800</Count>
     </item>
   </SALEvent>
+  <SALEvent>
+    <Subsystem>DIMM</Subsystem>
+    <EFDB_Topic>DIMM_logevent_lostContact</EFDB_Topic>
+    <item>
+      <EFDB_Name>device</EFDB_Name>
```

Comment:
If possible devices are know, it would be better to make event deviceStatus, with booleans (on/off) for connected/not connected devices.

Reply:
ok